docs/_includes/prevnext.html

```diff
@@ -0,0 +1,12 @@
+<div class="prevnext">
+  <div class="next">
+    {%- if include.nexturl -%}
+    <a href="{{include.nexturl}}">Next: {{include.nexttitle}} &gt;&gt;</a>
```

Comment:
Something wonky is going on here with escaping. see image:
![image](https://user-images.githubusercontent.com/5981958/47118821-772b2180-d21d-11e8-8202-e6af8cfa3baf.png)